scala/private/common.bzl

```diff
@@ -49,26 +46,19 @@ def _collect_jars_when_dependency_analyzer_is_on(dep_targets):
   runtime_jars = []
 
   for dep_target in dep_targets:
-    current_dep_compile_jars = None
-    current_dep_transitive_compile_jars = None
-
+    # we require a JavaInfo for dependencies
+    # must use java_import or scala_import if you have raw files
     if JavaInfo in dep_target:
       java_provider = dep_target[JavaInfo]
       current_dep_compile_jars = java_provider.compile_jars
       current_dep_transitive_compile_jars = java_provider.transitive_compile_time_jars
       runtime_jars.append(java_provider.transitive_runtime_jars)
-    else:
```

Comment:
Again the whole conditional as above but even if I’m wrong why silently ignore and not fail?